src/modules/SubmitScore.js

```diff
@@ -0,0 +1,16 @@
+async function SubmitScore(user, score, gameId) {
+    const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
```

Comment:
- [ ] Please 🙏 use `arrow functions` instead of the `function` keyword here as well

**Project requirements**

> Use `arrow functions` instead of the `function` keyword.